registers/TUTORIAL.md

```diff
@@ -0,0 +1,21 @@
+This tutorial will show you how to get started with Trillian, build a
```

Comment:
Given that this is a description of the current step rather than part of the code, I think it might be a better idea to move it to the pull request description.

That way it will serve its purpose more naturally and will also be already formatted nicely in Markdown by default.

Reply:
I am not planning to have these live forever as PRs. They're PRs so people can review them. :-)